interface/src/app/node/node_search.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * "adage.node_search" module.
+ */
+
+angular.module('adage.node_search', [
+  'ui.router',
+  'ui.bootstrap',
+  'ngResource',
+  'adage.mlmodel.components',
+  'adage.utils'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('node_search', {
+    url: '/node/search',
+    views: {
+      main: {
+        templateUrl: 'node/node_search.tpl.html',
+        controller: 'NodeSearchCtrl'
+        // See comments below on why "controllerAs" is not used.
+      }
+    },
+    data: {pageTitle: 'Node Search'}
+  });
+}])
+
+.factory('Node', ['$resource', 'ApiBasePath',
```

Comment:
I think there is some repetition between the code here and the 'NodeInfo' factory that already exists in the ``node.js`` file. Should we consolidate them into one?